BrainPortal/app/models/remote_resource.rb

```diff
@@ -490,7 +487,7 @@ def site
       host = "localhost"
       port = 3090+self.id  # see also in start_tunnels()
     end
-    "http://" + host + (port && port > 0 ? ":#{port}" : "") + dir
+    "http://" + ( host != nil ? host : "localhost" ) + (port && port > 0 ? ":#{port}" : "") + dir
```

Comment:
Why not

```ruby
    host.presence || "localhost"
```

which has the added side effect that a blank host is turned into localhost... ?